src/utils/errors.js

```diff
@@ -7,14 +10,20 @@ class ValidationError extends Error {
     if (this instanceof ContentTypeError) {
       this.message  = 'Incorrect content-type.'
       this.statusCode = 415
+      log.debug('Content-type validation failed. Will return 415 Unsupported Media Type.');
```

Comment:
Good call adding these!

Reply:
> Looks good! A few changes and questions. Also, could you talk a bit about the purpose of separating the DB and the dev DB? I am confused on the utility/usage of this

So I think the first advantage is that if we setup two environments, we can define the DB to use here by the env variable so we don't have to change the db config everytime.
Also I set the tests to use devDb, so we don't have to worry about forgetting to change the db config and accidentally running tests in production (I would very likely do this at some point lol).

Reply:
Makes sense to me! Should be useful in the future. Thanks!